project-communications/newsletter/newsletter-012921.md

```diff
@@ -0,0 +1,181 @@
+
+Best wishes for everyone!
+
+## Project News
+
+### Meet-up sessions
+
+We had our first open online meet-up session last Thursday (25th January). 
+This included a wonderful set of developers, researchers, and autistic collaborators with a whole variety of skills and perspectives. 
+Thank you everyone who took part! 
+
+The next meet-up session will take place on **Thursday 11th February** at an earlier time to accommodate more people. 
+Agenda and details to follow. 
+
+### We are on Twitter! 
+
+AutSPACEs now has a Twitter account! Follow us: https://twitter.com/AutSpaces!
+
+If you are social media savvy we would love to hear from you and get input into running the account, sharing it, and using it to promote the platform. 
+
+## Get involved
+
+Following the first meet-up group session on Thursday 25th January, a number of areas crucial to the project were identified in which people were interested in helping out:
+
+* Groups
+	* Research 
+	* Community engagement
+	* Promoting the platform 
+	* Web development  
+
+If you would like to join onee of these groups and meet and work with other autistic people, or to suggest another working group, please email Georgia at gaitkenhead@turing.ac.uk.
+
+### Take part in the ACCEPT study
+
+A team of clinicians, researchers and experts by experience are running a project to understand and share ways to help health, social care and public facing workforces across England to work with autistic people more effectively. 
+We want to help them make use of the Core Capabilities Framework for Supporting Autistic People (2019). 
+You can help by filling in the [ACCEPT survey](https://kclbs.eu.qualtrics.com/jfe/form/SV_cUxOb07WGQl6IO9?dm_i=4U0R,GIAA,2B1RTB,1Y9R6,1). 
+
+## Presenting the project
+
+The project is being presented on the **26th February from 2pm-3pm** for the *Social Data Science Spcecial Interest Group* at The Turing. 
+It is not a public event, but if you are interested in joining just email Georgia at gaitkenhead@turing.ac.uk and she will link you to the event. 
+
+## Introducing Alice
+
+Alice recently joined the Autistica/Turing citizen science project and has already launched herself into collaborating, co-facilitating our first online meet-up session and brainstorming strategies of engagement and areas of research for the project. 
+She is also founder, director, and CEO of a social purposes organisation based in Brighton and working across the South-East. 
+It works with young refugees and asymlum seekers. 
+Alice founded it almost 20 years ago. 
+The organisation is  called 'Pathways to Independence'. 
+It provides support and accommodation to young refugees and asylum seekers, unaccompanies 16+. 
+You can find our more on the organisation's [website](https://www.pathwaystoindependence.org.uk/).
+
+Here she is; 
+
+![](https://i.imgur.com/jJUd1uJ.jpg)
```

Comment:
Could potentially resize the image (https://stackoverflow.com/questions/24383700/resize-image-in-the-wiki-of-github-using-markdown) to make it more reader friendly?